pkg/util/util.go

```diff
@@ -29,3 +31,26 @@ func IsGPUVMI(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+func isIpv6Disabled() bool {
+	res, err := exec.Command("cat", "/proc/sys/net/ipv6/conf/default/disable_ipv6").Output()
```

Comment:
Please use a meaningful variable name - `res` can be anything. `ipv6Enabled` / `ipv6Disabled` clearly indicate what they are.

Reply:
done at https://github.com/kubevirt/kubevirt/pull/3379, see reason before as well please